### DISCUSSION TITLE
One.com: Cannot login to One.com server via Cyberduck

### DISCUSSION THREAD

**Discussion by Briefcoach** [iterate-ch/cyberduck#16022] [Q&A]
I follow the instructions from one.com, but it doesent work:
https://help.one.com/hc/en-us/articles/115005593865-How-do-I-connect-to-an-SFTP-server-with-Cyberduck

- and I've done everything it says, and when I try to connect in Cyberduck, it just hangs. - Any ideas about what I can do?

THANKS!

**Comment by tatze96** [comment]
Do you get an error message? If so, attach a screenshot.

Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), try to connect to the server, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-16022 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file and screenshot here, make sure to attach them through the GitHub web interface, as they wouldn't be visible to us otherwise.
